gpsdio_segment/segmenter.py

```diff
@@ -61,11 +61,8 @@ def __init__(
         self,
         instream,
         ssvid=None,
-        prev_msgids=None,
-        prev_locations=None,
-        prev_info=None,
         max_hours=Matcher.max_hours,
-        max_open_segments=20,
+        max_open_segments=100,
```

Comment:
No change requested, just a clarifying questions: Are there any potential ramifications to changing this metric? Would it cause segments to be slightly different in some cases because more of them remain open?

Reply:
@jennvanosdel , definitely.  What this mainly effects are the pathological MMSI (e.g., 412000000) that are highly spoofed.  This will do a bit better job assembling segments, since before it would immediately run out of open segments, however those MMSI still mostly end up defeating the segmenter.